handler/instance_started.go

```diff
@@ -268,8 +268,8 @@ func (h *InstanceStarted) TriggerImportDimensionOptions(r *recipe.Recipe, codeli
 		}
 
 		if r.Format == "cantabular_flexible_table" {
-			if cl.IsCantabularGeography != nil {
-				if *cl.IsCantabularGeography {
+			if cl.IsCantabularGeography != nil && cl.IsCantabularDefaultGeography != nil {
+				if *cl.IsCantabularGeography && !*cl.IsCantabularDefaultGeography {
```

Comment:
Doesn't need to be two conditionals, but it is a bit long on one line so dealer's choice.